client/src/api/v2/camera.ts

```diff
@@ -219,7 +223,7 @@ function createStatusPayloadHook<T>(
       newPayload: T | null,
     ) => setter(newPayload ?? initValue),
     returnHandler = (payload: T | null) => payload,
-  }: StatusPayloadOptions<T>,
+  }: StatusPayloadOptions<T, T>,
```

Comment:
This should be `U` as well since `createStatusPayloadHook` should also have the `<T, U>` generics.

Reply:
Doesn't line 225 `returnHandler = (payload: T | null) => payload` mean that `T` and `U` will always be the same?

Reply:
The `returnHandler = (payload: T | null) => payload` is just a default value. 
I.e. the default value of `returnHandler` is the identity function which takes a payload of type T.

Reply:
Gotcha, my bad